components/p3a/brave_p3a_scheduler.h

```diff
@@ -13,10 +13,9 @@ namespace brave {
 
 class BraveP3AScheduler : public metrics::MetricsScheduler {
```

Comment:
we probably can drop most of inheritance, because we are already far away from being similar to the parent classes

Reply:
we actively use functionality defined in the `MetricsScheduler`